SPECS/orangefs/orangefs.spec

```diff
@@ -188,13 +190,11 @@ install -p -m 644 %{SOURCE4} %{buildroot}%{_sysconfdir}
 %ldconfig_scriptlets
 
 %package devel
-
-Summary: Parallel network file system development libraries
-
-Requires: %{name}%{?_isa} = %{version}-%{release}
+Summary:        Parallel network file system development libraries
+Requires:       %{name}%{?_isa} = %{version}-%{release}
 
 %description devel
-%desc
+%{desc}
```

Comment:
Please move all `%package` and their `%description` sections between the default `%description` and the `%prep` section. That's the way we're trying to stick to.

Reply:
updated